Documentation/Signoffs/detaildesgincamera.md

```diff
@@ -0,0 +1,137 @@
+3. Maintainability
+    * Should require minimum maintenance once installed
+4. Power
+    * All cameras will be solar powered to provide minimal maintenance
+5. Communication
+    * Wireless communication
+        * Bluetooth not an option due to range	
+        * WiFi
+            * Need consideration of following factors:
+                * Quality
+                * Range
+                * Downtimes
+
+## Analysis
+
+1. Mounting
+    * In order for the camera to operate correctly it will need to be mounted securely onto the light post. However, a constraint we received from facilities is that drilling holes in the pole is not allowed. Due to this we need a mounting method that does not require drilling and is adjustable in the event that we need to relocate the camera on the pole. For this reason we plan to use a hose clamp/vertical bracket combination similar to Dahua Technology pole mount bracket (MFR# PFA150). Currently we have not received pole diameter dimensions from facilities so the correct hose clamp size has not been determined. However, this is a minor issue that can quickly be resolved by either measuring the pole or receiving the information from facilities. Hose clamps have a wide size range that should satisfy our requirements.
+    * The cameras shall be placed no less than 5 feet below the lights on the light posts to ensure that the emanating light is not obstructed at night. Although there is very little concern that the lights will affect the cameras during the system's hours of operation, the cameras shall be placed so that they do not obstruct the light at night.
+
+![Figure 1. Buildable PDF Schematic](../Images/Zummimal_Product_Diagram.png)]
+<h3>Figure 1. Buildable PDF Schematic</h3>
+<br />
+
+2. Monitoring
+    * The measurements for a single parking spot in the Bell Hall lot is 575 cm x 265 cm. Based on these measurements the whole area of the lot is then calculated in Figure 3.  The computer vision algorithms implemented [1] and [2] which detect and/or identify vehicles used training set images of the rear side of cars that were 64 x 64 pixels. A standard car is on average 5.8 feet [3], which translates to roughly a little over 10 pixels per foot (or 100 pixels per square foot) for these images of cars to be identified as such. Thus, the cameras must have a minimum pixels per foot of 10 pixels per foot (or 100 pixels per/square foot) to identify the presence of a car in a parking spot. Using 100 pixels/sq ft as a bare minimum for vehicle detection and mapping the area of the parking lot to be 23,261 sq ft. A total of 2.33 megapixels will be necessary for the whole lot. This includes the driving area, grass/shrubbery, and curbs around the area. Then, using the arrangement above, a total of nine cameras will be used. This suggests that each camera has a bare minimum resolution of 0.26 megapixels. The camera that we have chosen is 3 megapixels, which is above the required minimum.
+
+
```

Comment:
No analysis for if the camera will even be able to see spots when cars are parked. No analysis showing that this is the optimal allocation of cameras to minimize cost.